base/interactiveutil.jl

```diff
@@ -64,7 +64,27 @@ function edit(path::AbstractString, line::Integer=0)
     nothing
 end
 
-edit(f)          = edit(functionloc(f)...)
+function edit(f)
+    ms = methods(f)
+    nm = length(ms)
+    if nm <= 1
+        edit(functionloc(f)...)
+    else
+        for (i, m) in enumerate(ms)
+            @printf("%4u: %s\n", i, m)
+        end
```

Comment:
In the REPL I have up, `+` has 171 methods. While that is a few pagefulls, it's not even close to requiring an interrupt to stop the scrolling.

Maybe there might be a way to further enhance this functionality to provide for paging through the list when it is long, but that sounds more like functionality in the REPL proper. I'll have to think about that.